CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 - Disallow Basic Auth over HTTP in PowerShell Remoting on Unix (#6787)
 - Change packaging to differentiate only between major versions and previews (#6968)
 - Enhance and refactor `Add-Type` cmdlet (#6141) (Thanks @iSazonov!)
-  - A few error strings were removed and thus the corresponding fully-qualified error ids are not in use anymore.
+    + A few error strings were removed and thus the corresponding fully-qualified error ids are not in use anymore.
```

Comment:
Please use `    -` (4 spaces plus minus) to fix it. This is to keep consistent with the existing instances in this file.

Reply:
Ah I read the wrong rule in https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md.

Thought `mdspell` was complaining about [MD004](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md004---unordered-list-style).